integration-tests/src/test/java/tachyon/client/table/RawTableIntegrationTest.java

```diff
@@ -49,8 +48,8 @@ public final void after() throws Exception {
 
   @Before
   public final void before() throws Exception {
-    MasterContext.getConf().set(Constants.USER_FILE_BUFFER_BYTES, String.valueOf(100));
     mLocalTachyonCluster = new LocalTachyonCluster(10000, 1000, Constants.GB);
+    mLocalTachyonCluster.getTestConf().set(Constants.USER_FILE_BUFFER_BYTES, String.valueOf(100));
```

Comment:
This is from original code, and I think we should not freeride this PR for too much unrelated minor fix. We can probably create small tasks on this later.